src/main/java/dev/slimevr/vr/processor/skeleton/HumanSkeleton.java

```diff
@@ -164,10 +170,20 @@ public class HumanSkeleton extends Skeleton implements SkeletonConfigCallback {
 	protected ComputedHumanPoseTracker computedRightElbowTracker;
 	protected ComputedHumanPoseTracker computedLeftHandTracker;
 	protected ComputedHumanPoseTracker computedRightHandTracker;
+	// #endregion
+
+	// #region FK Settings
```

Comment:
I would prefer if settings like this are extracted out into a separate config class instead of being fields inline with the class they configure, because it helps reduce the large number of fields in this class. But thats out of scope of this PR.

Feel free to mark as resolved when you've read @Eirenliel